source/guides/stacks/dbrs.rst

```diff
@@ -0,0 +1,599 @@
+
+
+
+
+Firmware configuration
+**********************
+
+.. important::
+
+   When updating DCPMM FW, all DCPMM samples must be in the same mode (you cannot mix 1LM and 2LM samples).
+
+The latest firmware download for the Intel Server System S2600WF Family is avaliable at the `Intel Download Center`_
+
+Firmware Update Steps
+=====================
+
+#. Unzip the contents of the update package and copy all files to the root directory of a removable media (USB flash drive).
+#. Insert the USB flash drive to any available USB port on the system to be updated.
+#. Boot to EFI shell.
+#. Input "fsx(x:0,1,...):" to enter into your usb disk
+#. Run "startup.nsh"
+#. After update BMC firmware, system BIOS, ME firmware,FD, FRUSDR, system will reboot automatically.
+
+
+If Intel Optane DC Persistent Memory is installed, run startup.nsh a second time after the first reboot to upgrade Intel Optane DC Persistent Memory Firmware:
+
+* Boot to EFI shell.
+* Input "fsx(x:0,1,...):" to enter into your usb disk
+* Run "startup.nsh" again to update the corresponding AEP FW.
+
+
+Hardware Configuration
+**********************
+
+
+Online Resources
+================
+
+Before going through the configuration steps, we strongly recommend visiting the following resources and wikis to have a broader understanding of what is being done
+
+* `Quick Start Guide`_ Configure Intel Optane DC Persistent Memory Modules on Linux
+* `Managing NVDIMMs`_
+* `Configure, Manage, and Profile`_ Intel Optane DC Persistent Memory Modules
+
+Optane DIMM Configuration
+=========================
+
+The persistent memory DIMMs can be configured in devdax or fsdax mode. The use case to enable database stack on a kubernetes environment currently only support fsdax mode.
+
+Configuration Steps
+===================
+
+.. important::
+
+Run the following steps with root privileges (sudo) as shown in the examples
+
+
+#. To configure Optane DIMMs for App direct mode run this command and then reboot the system
+
+   .. code-block:: bash
+
+      sudo ipmctl create -goal PersistentMemoryType=AppDirect
+
+
+#. Next, list the pmem devices in the system
+
+   .. code-block:: bash
+
+      sudo ndctl list –N
+
+
+#. Create namespaces based on the regions and set mode as fsdax  -- use the names of the regions listed in previous step as the –-region parameter
+
+   .. code-block:: bash
+
+      sudo ndctl create-namespace --region=region0 --mode=fsdax
+
+
+#. Create the filesystem and mount it. We are using /mnt/dax{#} as a convention in this guide to mount our devices
+
+   .. code-block:: bash
+
+      sudo mkfs.ext4 /dev/pmem{n}
+      sudo mount -o dax /dev/pmem0 /mnt/dax0
+
+
+Running DBRS with Cassandra
+***************************
+
+DBRS with Cassandra can be deployed as a standalone container or inside Kubernetes. Instructions for both cases is included here. Note that you can use the released `Docker image`_.  These instructions are to provide a baseline for creating your own container image.
```

Comment:
Add a corresponding URL at bottom for ``Docker image``_.

Reply:
Added URL but it's blank until release is official.

Reply:
And in next commit fixed the whitespace I put in the link name by mistake...